tests/api/project/schemas/test_updatable_model.py

```diff
@@ -223,10 +223,10 @@ class ChildModel(ParentModel):
 
 def test_updatable_model_with_sub_classes_and_template_values_and_custom_validator_in_child():
     class ParentModel(UpdatableModel):
-        a: str
+        a: str = Field()
 
     class ChildModel(ParentModel):
-        b: str
+        b: str = Field()
```

Comment:
This is what I was talking about earlier. Field is not required, and it might make it harder for developers. Might make sense to skip fields starting with Underscore or if we can figure out they are PrivateType